File: 1201-1300/1295-find-numbers.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        result = 0
        for num in nums:
            # convert int to str to get the length(digit)
            # of the number
            # even number is divisble by 2 without any
            # leftover
            if len(str(num)) % 2 == 0:
                result += 1
        return result

File: 1201-1300/1295_find_numbers.go
func findNumbers(nums []int) int {
	res := 0

	for _, num := range nums {
		digit := 0
		for num > 0 {
			num /= 10
			digit++
		}

		if digit%2 == 0 {
			res++
		}
	}

	return res
}